### ISSUE TITLE
exec path error

### ISSUE THREAD

**Issue by cpapidas** [vweevers/jscript#1]
when i try to run the script from another dir i have the following error 

Uncaught Error: Command failed: C:\WINDOWS\Sysnative\cscript.exe //Nologo //B D:\Project\...\public\app\config\ActiveXObjects.js

e.g.
node another_dir/exec.js {error}

node exec.js {works fine}

**Comment by vweevers** [comment]
`jscript` resolves the script from the working directory. I'm guessing you need:

```js
jscript(path.join(__dirname, 'script.js'))
```

But it's hard to tell without knowing the contents of `exec.js`. Can you elaborate a little?

**Comment by cpapidas** [comment]
For some reason the module does not work for me. I have the following files:

file1.js
```
var JSON = require('json3')
while(!WScript.StdIn.AtEndOfStream) {
    var file = {version: ""};
    file.version = "test message";
    WScript.StdOut.Write(JSON.stringify(file))
}
```


exec.js
```
var jscript = require('jscript');
var duplex = jscript('file1.js', { json: true })
duplex.on('data', function(file) {
    console.log("-->", file)
});
duplex.end();
```
execute with:

`$ node exec.js`


**All i want is to pass a message from file1.js to exec.js**

**I DON'T HAVE ANY ERROR NEITHER ANY CONSOLE LOG MESSAGE :/**

**Comment by vweevers** [comment]
A couple of points:

- You can drop the while loop if you're not using stdin:

```js
var JSON = require('json3')
var file = {version: ""};
file.version = "test message";
WScript.StdOut.Write(JSON.stringify(file))
```

- But if you are planning to use stdin, then be sure to add a newline between JSON chunks:

```
var JSON = require('json3')
while(!WScript.StdIn.AtEndOfStream) {
    var file = {version: ""};
    file.version = "test message";
    WScript.StdOut.Write(JSON.stringify(file) + "\n") // important
}
```

- To to be able to use `require`, you must first transform the code with [jscriptify](https://www.npmjs.com/package/jscriptify):

```
npm install jscriptify --global
jscriptify file1.js > bundle.js
```

Then change `exec.js` to use that bundle:

```js
var duplex = jscript('bundle.js', { json: true })
```

- To be able to call `exec.js` from a directory other than the working directory, add this:

```js
var path = require('path');
var bundle = path.join(__dirname, 'bundle.js'); // make it an absolute path
var duplex = jscript(bundle, { json: true })
```

- Lastly, I discovered that `jscript` is incompatible with node 6. I will push an update shortly.

**Comment by vweevers** [comment]
Published 2.0.0 with support for Node 6 and 7.

**Comment by cpapidas** [comment]
Thank you very much, my problem was that i used the require without jscriptify.
